.github/workflows/tests.yml

```
@@ -27,30 +24,15 @@ jobs:
 
     - name: Set up Homebrew
       run: |
+        cd $(brew --repo)
+        git fetch --tags origin ${{github.sha}}
+        git checkout --force FETCH_HEAD
+        git branch -f master HEAD
```

```suggestion
        git checkout -B master FETCH_HEAD
```

or 

```suggestion
        git checkout --force -B master FETCH_HEAD
``` 

if it's required (I'm pretty sure it's not)

or even more robust perhaps:

```suggestion
        git checkout -B master ${{github.sha}}
```

---

If I recall, this doesn't work because you can't force-checkout a branch that you're currently on.

---

Works fine here:

```
mikebook # git checkout -B master HEAD^
Reset branch 'master'
Your branch is behind 'origin/master' by 6 commits, and can be fast-forwarded.
  (use "git pull" to update your local branch)
```